whatwg-streams/index.d.ts

```diff
@@ -124,11 +124,11 @@ declare class WritableStreamDefaultController {
 declare class ByteLengthQueuingStrategy {
     constructor({ highWaterMark }: { highWaterMark: number });
 
-    size(chunk: ArrayBufferView): number;
+    size(chunk: ArrayBufferView): number | undefined;
```

Comment:
See https://streams.spec.whatwg.org/#blqs-size
> (If the chunk doesn’t have one, it will return undefined, causing the stream using this strategy to error.)